README.md

```diff
@@ -54,14 +54,18 @@ jobs:
 
 The action knows to exclude its own check run (i.e. the GitHub Actions job that runs `wait-for-status-check`)
 
+When some of your checks trigger with a delay (e.g. workflow is using `concurrency` and has to wait for previous to be cancelled), then correctly setup `delay` parameter. The action will wait given amount of seconds to let your checks start.
```

Comment:
I think we could do without this line, delay should be pretty intuitive without mentioning your specific scenario

Reply:
Removed :)